### PR TITLE
Use black formatter

### LINKED ISSUE #28: PEP8 style guide
As a python developer and a big fan of podman, I might find this package useful in order to automate some container processes. But now, I see there's a lack of features and I think this is a perfect moment to improve code quality. [PEP8](https://www.python.org/dev/peps/pep-0008/) is the official style guide for python and this library could follow those principles in a more strictly way.

i. e.

Before PEP8
```python
def __init__(self, url, base="/v1.24/libpod", *args,
    **kwargs):  # pylint: disable-msg=W1113
```

After PEP8
```python
def __init__(
    self, url, base="/v1.24/libpod", *args, **kwargs
):  # pylint: disable-msg=W1113
```

### PR DESCRIPTION
**Pull request by mwhahaha**
This change documents the usage of black as the prefered formatter for
this project. Additionally it adds two tox targets that can be used to
have black be invoked.

* `tox -e black` will print out what would be changed
* `tox -e black-format` will actually update the files with the code
  formatted

Black is configured via pyproject.toml with a line-length of 100 and
will skip string normalization

Closes #28

Signed-off-by: Alex Schultz <aschultz@redhat.com>

### CODE REVIEW

**Comment by mwhahaha** [comment]
Should hold off on this until #38 lands, then it'll just be a tox env and documentation

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,45 +1,75 @@
...
+- [pylint](https://www.pylint.org/)
+- [black](https://github.com/psf/black)
+- [tox](https://tox.readthedocs.io/en/latest/)
+- You may need to use [virtualenv](https://virtualenv.pypa.io/en/latest/) to
+  support Python 3.6
+
 ## Testing
-Depending on the size of your PR we will expect at a minimum unit tests. 
+
+Depending on the size of your PR we will expect at a minimum unit tests.
```
Comment (jwhonce) [inline CONTRIBUTING.md:37]: @mwhahaha Thanks for this work.

Given your coverage work and we're in the high 80's on coverage. How about putting a number on the unit test/coverage results? Won't merge if PR coverage falls below 85%?

Comment (mwhahaha) [reply CONTRIBUTING.md:37]: Sure we can raise that and document it

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**